client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -101,6 +101,9 @@
     @Option(names = "--password", paramLabel = "<password>", description = "Prompt for password")
```

Comment:
Can you please write in commit message how one can test it locally.

Reply:
ok